### ISSUE TITLE
Ability to customize naming conventions

### ISSUE THREAD

**Issue by Mahesh-MSFT** [Azure/Enterprise-Scale-for-AVS#150]
customers are not being able to use AVS landing zone accelerator because it does not provide them ability to specify their unique naming conventions.

Resources such as resource group, VNet name, subnet name, etc. are getting created by a pre-defined naming convention hard-coded in ARM template. Some examples below.

```
"resources": [
            {
              "type": "Microsoft.Resources/resourceGroups",
              "apiVersion": "2021-04-01",
              "name": "[format('{0}-PrivateCloud', parameters('Prefix'))]",
              "location": "[parameters('Location')]"
            },
....

"resources": [
                    {
                      "type": "Microsoft.AVS/privateClouds",
                      "apiVersion": "2021-06-01",
                      "name": "[format('{0}-SDDC', parameters('Prefix'))]",

....
 "resources": [
                    {
                      "type": "Microsoft.Network/virtualNetworks/subnets",
                      "apiVersion": "2021-02-01",
                      "name": "[format('{0}/{1}', parameters('VNetName'), 'JumpBox')]",
```
 
Enabling customers to completely customize names assigned to resources will encourage them to use AVS LZA.

**Comment by prasad3017** [comment]
@shaunjacob  addressed this with PR #155 please check and could you please run with the customer

**Comment by prasad3017** [comment]
@Mahesh-MSFT  This is available now
